dapp/src/components/buySell/SellWidget.js

```diff
@@ -73,6 +76,35 @@ const SellWidget = ({
     .filter((coinSplit) => parseFloat(coinSplit.amount) > 0)
     .map((coinSplit) => coinSplit.coin)
 
+  const priceToleranceLocalStorageKey = 'selected_redeem_price_tolerance'
+
+  const setPriceToleranceValue = (value) => {
+    setPriceTolerFct(value)
+    localStorage.setItem(priceToleranceLocalStorageKey, value)
+  }
+  const stableCoinSplitsSum = sellWidgetCoinSplit
+    .map((split) => parseFloat(split.amount))
+    .reduce((a, b) => a + b, 0)
+
+  const exitFee = ousdToSellNumber * redeemFee
+  const exchangeRateLoss = ousdToSellNumber - stableCoinSplitsSum - exitFee
+  const expectedStablecoins = stableCoinSplitsSum
+  const minStableCoinsReceived =
+    priceToleranceValue && ousdToSellNumber
+      ? expectedStablecoins - (expectedStablecoins * priceToleranceValue) / 100
+      : 0
+
+  // store price tolerance value for future default setting
+  useEffect(() => {
+    // default price tolerance value
+    let priceTolerance = 0.5
+    let localStorageValue = localStorage.getItem(priceToleranceLocalStorageKey)
+    if (localStorageValue) {
+      priceTolerance = parseFloat(localStorageValue)
+    }
+    setPriceToleranceValue(priceTolerance)
+  })
```

Comment:
Thinking about it, this could very well be moved into its own hook, something like `usePriceTolerance`

Reply:
I like that! moving to a hook